Frontend/ui-customer/src/MerchantConfigs/YatriConfig.js

```diff
@@ -0,0 +1,41 @@
+export function getMerchantString(key){
+    var selectedLanguage = JBridge.getKeysInSharedPrefs("LANGUAGE_KEY");
+    switch(selectedLanguage) {
+        case "ML_IN" :
+            return getStringMLValue(key);
+        default :
+            return getStringENValue(key);
+      }
+}
+
+export function isMerchantString(key){
+    return config.StringKeys.includes(key);
+}
+
+export function getStringENValue(key){
+    if (key in englishStrings){
+        return englishStrings[key];
+    }
+    return "error in getCommonEN";
+}
+
+export function getStringMLValue(key){
+    if (key in malayalamStrings){
+        return malayalamStrings[key];
+    }
+    return "error in getCommonKN";
+}
+
+
+const config = {
+    StringKeys : ["WELCOME_TEXT", "ABOUT_TEXT"]
```

Comment:
we can have similar way of looking up into keys in customer and driver

Reply:
not sure which is best.needs discussion